config/config.yaml

```diff
@@ -6,7 +6,7 @@ server:
     key:
   edition: local-ce:dev
   usage:
-    enabled: true
+    enabled: false
```

Comment:
Any reason to turn it off here? I remember you mentioned that the usage client crashes the service under certain circumstances, but I think it should be normal most of the time.